index.html

```diff
@@ -271,13 +273,48 @@ <h2>On behalf of organisations</h2>
 
 			</article>
 
+			<article class='container' lang='pt'>
+
+				<p>Nós somos uma comunidade de indivíduos que têm interesse significativo no desenvolvimento e na saúde da World Wide Web (“a Web”). <strong>Nós estamos profundamente preocupados</strong> sobre o Accelerated Mobile Pages (“AMP”), um projeto do Google que supostamente procura melhorar a experiência do usuário na Web.</p>
+
+				<p>Na verdade, a AMP mantém os usuários no domínio do Google e desvia o tráfego de outros sites para benefício do Google. Em uma escala de bilhões de usuários, isso tem o efeito de <strong>reforçar ainda mais o domínio do Google na Web.</strong></p>
+
+				<p>Reconhecemos o problema das páginas da Web serem lentas para carregar, em relação a tecnologias alternativas, como <a href='https://instantarticles.fb.com/'>Facebook Instant Articles</a> e <a href='https://developer.apple.com/news-publisher/'>Apple News</a>. Os editores (especialmente nos meios de comunicação) já enfrentaram escolhas difíceis e pouco incentivos, levando a decisões ruins e por consequência uma terrível experiência para os usuários.</p>
```

Comment:
“Poucos incentivos” is less jargon-y than “maus incentivos”, but the original meaning is closer to “incentivados a tomar decisões ruins”.